src/js/services/utilService.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
```

Comment:
Yes, that 's the idea, because right now we haven't any common service of utilities. Maybe we need it later.